modules/presets/bitcoind-remote.nix

```diff
@@ -22,4 +22,7 @@ in {
       RemainAfterExit = true;
     };
   };
+
+  # Disable banlist loading
+  systemd.services.bitcoind-import-banlist = lib.mkForce {};
```

Comment:
Why disable the banlist specifically in this preset?

Reply:
I've removed this.
My original reasoning was that client banlist loading is redundant in a scenario where multiple nix-bitcoin clients connect to a nix-bitcoin server.
But it's still relevant in case of a non nix-bitcoin server.